.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
According to the requirements, you've installed a stage-2 preset.
I am not a big fan of that, but still either use it in .babelrc or remove it at all – both options are fine.